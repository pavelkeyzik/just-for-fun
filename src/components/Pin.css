.Pin__icon {
  fill: var(--pink);
}
