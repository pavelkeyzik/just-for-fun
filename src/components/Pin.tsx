import React from 'react';

import { ReactComponent as PinIcon } from '../images/pin.svg';

import styles from './Pin.module.css';

export function Pin(): JSX.Element {
  return <PinIcon className={styles.pin} />;
}
