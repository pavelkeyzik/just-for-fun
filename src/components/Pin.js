import React from 'react';

import { ReactComponent as PinIcon } from '../images/pin.svg';

import './Pin.css';

export function Pin() {
  return <PinIcon className="Pin__icon" />;
}
