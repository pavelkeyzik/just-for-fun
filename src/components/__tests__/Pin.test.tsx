import React from 'react';
import { render } from '@testing-library/react';
import { Pin } from '../Pin';

test('Should render SVG element with Pin icon', () => {
  const { container } = render(<Pin />);

  expect(container.firstChild).toMatchSnapshot();
});
