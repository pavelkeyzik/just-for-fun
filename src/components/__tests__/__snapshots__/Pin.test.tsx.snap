// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render SVG element with Pin icon 1`] = `
<svg
  class="pin"
  data-file-name="SvgPin"
/>
`;
